File: 1281. Subtract the Product and Sum of Digits of an Integer/Solution.cpp
class Solution {
public:
    int subtractProductAndSum(int n) {
        int sum1 = 0;
        int sum2 = 1;
        while (n)
        {
            int digit = n % 10;
            sum1 += digit;
            sum2 *= digit;
            n /= 10;
        }
        return sum2 - sum1;
    }
};

File: 1281. Subtract the Product and Sum of Digits of an Integer/Solution.py
class Solution:
    def subtractProductAndSum(self, n):
        """
        type n: int
        rtype: int
        """
        sum1 = 0
        sum2 = 1
        while n:
            digit = n % 10
            sum1 += digit
            sum2 *= digit
            n = n // 10
        return sum2 - sum1